BundesIdent/Views/Setup/SetupPersonalPIN/SetupPersonalPINState.swift

```diff
@@ -64,21 +73,28 @@ enum SetupPersonalPINAction: BindableAction, Equatable {
 let setupPersonalPINReducer = Reducer<SetupPersonalPINState, SetupPersonalPINAction, AppEnvironment> { state, action, environment in
     switch action {
     case .onAppear:
-        return .none
+        let effect = Effect<SetupPersonalPINAction, Never>(value: .focus(.pin1))
+        if #available(iOS 16, *) {
+            return effect
+        } else {
+            // On iOS 15, setting a focus state only works after a short delay
+            return effect.delay(for: 0.5, scheduler: environment.mainQueue).eraseToEffect()
```

Comment:
We use 0.5 seconds here and in View+FocusOnAppear we use 0.75. Could we use only one of those numbers and even make that a constant?